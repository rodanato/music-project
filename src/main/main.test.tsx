

import React from 'react';
import renderer from 'react-test-renderer';
import MainOrganism from './main.organism';
import HeaderOrganism from './header/header.organism';
import SliderOrganism from './slider/slider.organism';
import FooterOrganism from './footer/footer.organism';
import NavigationOrganism from './navigation/navigation.organism';

it('MainOrganism snapshot example', () => {
  const tree = renderer.create(
    <MainOrganism
      header={<HeaderOrganism/>}
      slider={<SliderOrganism/>}
      footer={<FooterOrganism/>}
      navigation={<NavigationOrganism/>}
    >
    </MainOrganism>
    ).toJSON();
  expect(tree).toMatchSnapshot();
});