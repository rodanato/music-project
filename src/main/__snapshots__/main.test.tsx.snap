// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainOrganism snapshot example 1`] = `
<main
  className="css-ho9uo1"
>
  <div
    className="css-1224u6r"
  >
    <div
      className="css-gu1zkw-MainOrganism"
    >
      <section
        className="css-8atqhb"
      >
        Header
      </section>
    </div>
    <div
      className="css-npnykh"
    >
      <section
        className="css-djz5fk-SliderOrganism"
      >
        <section
          className="css-cbl75v"
        >
          Data
        </section>
        <section
          className="css-uq7dtg"
        >
          Content
        </section>
        <section
          className="css-b95f0i"
        >
          <div
            className="mpp-font--text css-1op49o9-MenuItemAtom"
          >
            Albums
          </div>
          <div
            className="mpp-font--text css-1op49o9-MenuItemAtom"
          >
            Artists
          </div>
          <div
            className="mpp-font--text css-1op49o9-MenuItemAtom"
          >
            Songs
          </div>
        </section>
      </section>
      <section
        className="mpp-font--bold css-18opcsq"
      >
        Navigation
      </section>
    </div>
    <div
      className="css-sd7lsn"
    >
      <section
        className="css-8atqhb"
      >
        Footer
      </section>
    </div>
  </div>
</main>
`;
